tests/test_actions.py

```diff
@@ -174,7 +214,20 @@ def testCalcBinnedStats(self):
         for key, value in truth.items():
             self.assertEqual(result[key], value, key)
 
-    # def testCalcRhoStatistics(self): TODO: implement
+    def testCalcRhoStatistics(self):
+        # just check if runs
+        rho = CalcRhoStatistics()
+        rho.treecorr.nbins = 21
+        rho.treecorr.min_sep = 0.01
+        rho.treecorr.max_sep = 100.0
+        rho.treecorr.sep_units = "arcmin"
+        result = rho(self.dataRhoStats, band="r")
+        for rho in result:
+            if rho != "rho3alt":
+                assert np.sum(np.isfinite(result[rho].xip)) == len(result[rho].xip)
+                assert np.sum(np.isfinite(result[rho].xim)) == len(result[rho].xim)
+            else:
+                assert np.sum(np.isfinite(result[rho].xi)) == len(result[rho].xi)
```

Comment:
```suggestion
                self.assertEqual(np.isfinite(result[rho].xi)), len(result[rho].xi))
```

and similar for the ones above, so you get a more helpful error message if/when the test fails.

Reply:
Or, `self.assertEqual(len(result[rho].xi), rho.treecorr.nbins)` and `self.assertTrue(np.isfinite(result[rho].xi).all())`